docs/source/API/core/parallel-dispatch/single.md

```diff
@@ -0,0 +1,84 @@
+# `fence()`
+
+Header File: `Kokkos_Core.hpp`
+
+Usage:
+
+```c++
+In a parallel dispatch statement, execute a lambda once per Thread or Team.
+- Kokkos::single` accepts two policies:
+  - Kokkos::PerTeam restricts execution of the lambda's body to once per team
+  - Kokkos::PerThread restricts execution of the lambda body to once per thread (i.e., to only one vector lane in a thread).
+
+```
+
+The `single` function can take two arguments:  an execution `Policy` and a `Lambda`.  `Kokkos::single` is only valid for the `Lambda` to capture variables by value (i.e., [=], and not [&]). That `Lambda` takes zero arguments or one argument by reference.  If `Lambda` takes one argument, the final value of that argument is broadcast to every executor, for example, every vector lane of the Thread, or every Thread (and vector lane) of the Team. 
+
+
+## Interface
+
+```c++
+Kokkos::single(Policy,Lambda);
+```
+
+### Parameters
+
+- `Policy`: Defines how the execution will be performed (in the context of a parallel dispatch statement)..
```

Comment:
I would either remove the list of one item or add the second argument and an explanation for it, even though it might seem obvious.